packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -2,6 +2,21 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
+import 'dart:convert' show JSON;
+import 'dart:developer' as developer;
+
+import 'package:meta/meta.dart';
+
+import 'assertions.dart';
+import 'basic_types.dart';
+
+/// Signature for service extensions.
+///
+/// The returned map must not contain the keys "type" or "method", as they will be
+/// replaced before the value is sent to the client.
```

Comment:
I don't. This must have been something specific to the extension using them. Driver responses, for example, do not have "type" and "method" in the response.